tests/func_test.py

```diff
@@ -149,17 +148,13 @@ def test_add_teg(self):
         tag_input.send_keys('test tag')
         submit_button.submit()
 
+        element_locator = (By.XPATH, '//button[@id="tag-test tag"]')
         WebDriverWait(self.browser, 10).until(
-            EC.presence_of_element_located((By.XPATH, '//button[@id="tag-test tag"]')))
-        max_attempts = 10
-        while max_attempts > 0:
-            try:
-                tag_text = self.browser.find_element(By.XPATH, '//button[@id="tag-test tag"]').text
-                self.assertIn('test tag', tag_text)
-                break
-            except StaleElementReferenceException:
-                max_attempts -= 1
-                time.sleep(1)
```

Comment:
Oh, I see

You have removed this...
Please ignore the comment in the previous commits related to these time.sleep()s